app/assets/stylesheets/config/_variables.scss

```diff
@@ -111,7 +111,7 @@
   --z-elevate: 1; // to elevate something, just a little, in relation to its sibling
   --z-sticky: 100; // header, mobile actions bar
   --z-drawer: 200; // sidebar sliding nav drawers
-  --z-modal: 300; // modals ¯\_(ツ)_/¯
   --z-dropdown: 400; // all kinds of dropdowns
+  --z-modal: 500; // modals
```

Comment:
Updated since I'm pretty sure we always want modals to appear at the highest elevation (and over the top of any dropdown)